internal/zinc-scripted/src/test/scala/sbt/internal/inc/IncHandler.scala

```diff
@@ -414,17 +439,24 @@ case class ProjectStructure(
 
   def packageBin(i: IncInstance): Unit = {
     compile(i)
-    val jar = targetDir / s"$name.jar"
-    val manifest = new Manifest
-    val sources =
-      (classesDir ** -DirectoryFilter).get flatMap {
-        case x =>
-          IO.relativize(classesDir, x) match {
-            case Some(path) => List((x, path))
-            case _          => Nil
+    val targetJar = targetDir / s"$name.jar"
+    outputJar
```

Comment:
Can we replace `map` and `getOrElse` by a simple `match` expression? (This comment also applies to other code I've seen in this PR)

Reply:
Yeah, sometimes I just forget that I can use pattern matching and it ends up unreadable. I think you meant the `map + getOrElse` in `Analyzer`. Fixed.